scripts/deployed_general_agent_viewer.py

```diff
@@ -57,10 +68,22 @@ def starting_balance(self) -> float:
 )
 st.title("Deployed Trader Agent Viewer")
 
-check_required_api_keys(["BET_FROM_PRIVATE_KEY"])
```

Comment:
`check_required_api_keys` not needed anymore, because now the requried env vars aren't optional in  DeployedGeneralAgentSettings